onnxruntime/core/framework/tensorprotoutils.h

```diff
@@ -52,7 +52,11 @@ common::Status TensorProtoToMLValue(const Env& env, const ORTCHAR_T* tensor_prot
  * @param tensor_proto  source data
  * @param tensorp       destination empty tensor
  * @return
-*/
+ */
+common::Status TensorProtoToTensor(const Env& env, const ORTCHAR_T* model_path,
+                                   const ONNX_NAMESPACE::TensorProto& tensor_proto,
+                                   Tensor& tensor, OrtCallback&);
+
 common::Status TensorProtoToTensor(const Env& env, const ORTCHAR_T* model_path,
                                    const ONNX_NAMESPACE::TensorProto& tensor_proto,
                                    Tensor& tensor);
```

Comment:
Please add the documentation comment for the additional TensorProtoToTensor function.

The function with OrtCallback needs to have that parameter documented. That should include info on the special case where external data is directly referenced if the callback has a deleter function.